tests/operations/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -104,4 +105,5 @@ $a . $b;
 $a .= $b;
 $a instanceof stdClass;
 $output = \`ls -al\`;
+$a = (boolean) "test";
```

Comment:
(bool) got turned into (boolean)?

Reply:
I think both are acceptable - http://php.net/manual/en/language.types.type-juggling.php. do people have thoughts on enforcing one convention or reusing what was originally written?

Reply:
We have the same situation for flow and we normalize it to one with prettier

Reply:
both are acceptable for casting, but it seems only the short form is valid for type hints, so it may be better to use the short form also when casting ?

http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration

PSR 12 seems to lean that way: https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#25-keywords-and-types
`Short form of type keywords MUST be used i.e. bool instead of boolean, int instead of integer etc.`

Reply:
I agree, always using the short form seems most clear and consistent. I opened a PR in the parser, i think it should be fixed there: https://github.com/glayzzle/php-parser/pull/102